CMakeLists.txt

```diff
@@ -22,15 +22,20 @@ set(CMAKE_SKIP_RPATH TRUE)
 set(MCM_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(MP_DIR ${CMAKE_CURRENT_SOURCE_DIR}/media-proxy)
 set(SDK_DIR ${CMAKE_CURRENT_SOURCE_DIR}/sdk)
+set(TESTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests/unit)
 set(SDK_INCLUDE_DIR ${SDK_DIR}/include)
 
 # Setup output folder
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
+enable_testing()
+
 add_subdirectory(${SDK_DIR})
 add_subdirectory(${MP_DIR})
+add_subdirectory(${TESTS_DIR})
```

Comment:
I think a flag for enable/disable tests is mandatory. Could you add it?

Reply:
I've add the flag, but set it to build the test by default.